tests/test_data/output_step1.smi

```diff
@@ -1,1976 +1,1976 @@
-C=CC1(C)CCc2c3c(cc(O)c2C1O)C(C)(C)C(O)CC3
-CCc1cc(S(=O)(=O)NCCCCO)ccc1-c1ccc(F)cc1F
-O=C(c1ccc(OCc2ccc(CN3CCCCC3)cc2)cc1)C1CC1
-O=C(O)CN1CCCC(=C2N(CC(=O)O)c3ccccc3N2C2CC3CCCC(C2)N3C2CC3CCCCC(C3)C2)C1
-CCN(CC)C(=O)c1c(NC(=O)c2cccs2)sc2c1CCC2
-Cc1cn(-c2ccn3c(-c4cccc(NC(=O)NCC(F)(F)F)c4)cnc3c2)cn1
-Cn1nccc1-c1nc(Nc2n[nH]c3ccccc23)c2c(n1)C(C)(C)OC2
 O=C(Cc1c[nH]c2ccccc12)NN=Cc1ccco1
```

Comment:
Let's see if we can use the file from before and still have everything pass.

Reply:
Reverted it back to the original one.